src/index.css

```diff
@@ -14,7 +9,7 @@
 a {
   font-weight: 500;
   color: #646CFF;
```

Comment:
We can use for the links the font color of their parent bt [inherit](https://developer.mozilla.org/en-US/docs/Web/CSS/inherit)
```suggestion
  color: inherit;
```

As on design 
![image](https://github.com/3apy6a/fe-react-2024/assets/98031768/66746526-f399-43d9-9c76-67e992b93387)

Reply:
mach better :)

Reply:
done